tests/gold_tests/remap/remap_http.test.py

```diff
@@ -28,18 +28,24 @@
 # Define default ATS
 ts = Test.MakeATSProcess("ts")
 server = Test.MakeOriginServer("server")
+server2 = Test.MakeOriginServer("server2",lookup_key="{%Host}{PATH}")
```

Comment:
`lookup_key`? Let it slide for now, but gosh that's confusing compared to the command line args.